aws/cleanup.py

```diff
@@ -50,7 +50,7 @@ def default_ctor(_dummy, tag_suffix, node):
 
     yaml.add_multi_constructor('', default_ctor)
 
-    with open(config_path) as config_fd:
+    with open(config_path, encoding="UTF-8") as config_fd:
```

Comment:
I rarely ever see the encodings uppercased in Python. I suppose, let's stick to the convention:
```suggestion
    with open(config_path, encoding='utf-8') as config_fd:
```